libnetwork/netutils/utils.go

```diff
@@ -124,6 +124,9 @@ func GenerateMACFromIP(ip net.IP) net.HardwareAddr {
 // GenerateRandomName returns a new name joined with a prefix.  This size
 // specified is used to truncate the randomly generated value
 func GenerateRandomName(prefix string, size int) (string, error) {
+	if size > 64 {
+		panic("size cannot be greater than 64")
```

Comment:
A more descriptive message may be preferable. I will update it to include the invalid value. It may be worth also checking for negative numbers.

Reply:
A more descriptive message has been added, which includes the size